plugins/cua/vendor/cua-driver/source/Tests/integration/test_hermes_chrome_form_fill.py

```
@@ -0,0 +1,603 @@
+        self._losses_before       = _read_file_int(_LOSS_FILE)
+        self._key_losses_before   = _read_file_int(_KEY_LOSS_FILE)
+        self._field_losses_before = _read_file_int(_FIELD_LOSS_FILE)
+
+    def _run_hermes(self, prompt: str, timeout: int = 120, max_turns: int = 5) -> str:
+        """Run hermes with the anthropic provider and return combined stdout+stderr."""
+        env = os.environ.copy()
+        env["ANTHROPIC_API_KEY"] = _ANTHROPIC_KEY
+        env["HERMES_COMPUTER_USE_BACKEND"] = "cua"
+        env["HERMES_CUA_DRIVER_CMD"] = default_binary_path()
+
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)